tests/components/systemmonitor/test_psi_sensor.py

```diff
@@ -0,0 +1,92 @@
+"""Test System Monitor PSI sensor."""
```

Comment:
Include additional tests in the already existing `test_sensor.py` file, don't create another test file.